tests/spread/rockcraft/extension-expressjs/task.yaml

```diff
@@ -0,0 +1,72 @@
+summary: expressjs extension test
+environment:
+  SCENARIO/bare: bare
+  SCENARIO/base_2404: ubuntu-24.04
+  ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: "true"
+
+execute: |
+  NAME="expressjs-${SCENARIO//./-}"
+  ROCK_FILE="${NAME}_0.1_amd64.rock"
+  IMAGE="${NAME}:0.1"
+
+  sudo apt-get update -y && sudo apt-get install npm -y
+  npm install -g express-generator@4
+  express app
+  cd app && npm install && cd ..
+
+  run_rockcraft init --name "${NAME}" --profile expressjs-framework
+  sed -i "s/^name: .*/name: ${NAME}/g" rockcraft.yaml
```

Comment:
weird, shouldn't the name be right already (because of `--name`)?
Can you confirm whether this is a bug in rockcraft?

Reply:
You're right, removed!